src/vs/workbench/parts/search/electron-browser/search.contribution.ts

```diff
@@ -643,6 +643,11 @@ configurationRegistry.registerConfiguration({
 			],
 			default: 'auto',
 			description: nls.localize('search.collapseAllResults', "Controls whether the search results will be collapsed or expanded."),
+		},
+		'search.useReplacePreview': {
+			type: 'boolean',
+			default: true,
+			description: nls.localize('search.useReplacePreview', "Controls whether to open Replace Preview when replacing a match."),
```

Comment:
Not just when replacing, but when clicking a match.

Reply:
You are right I'll reword this.